core/tests/protractor_desktop/embedding.js

```diff
@@ -104,6 +104,31 @@ describe('Embedding', function() {
     await workflow.publishExploration();
   };
 
+  var EMBEDDING_ERRORS_TO_IGNORE = [
+    'http:\/\/localhost:9001\/assets\/scripts\/' +
+    'embedding_tests_dev_0.0.2.min.html - Refused to display ' +
+    '\'http:\/\/localhost:9001\/embed/exploration\/idToBeReplaced',
+    'http:\/\/localhost:9001\/assets\/scripts\/' +
+    'embedding_tests_dev_0.0.2.min.html - Refused to display ' +
+    '\'http:\/\/localhost:9001\/embed\/exploration\/idToBeReplaced\\?' +
+    'locale=en#version=0.0.2&secret=',
+    'http:\/\/localhost:9001\/assets\/scripts\/' +
+    'embedding_tests_dev_0.0.2.min.html - Refused to display ' +
+    '\'http:\/\/localhost:9001\/embed\/exploration\/fake_id\\?' +
+    'locale=en#version=0.0.2&secret=',
+    'http:\/\/localhost:9001\/assets\/scripts\/' +
+    'embedding_tests_dev_i18n_0.0.1.min.html - Refused to display ' +
+    '\'http:\/\/localhost:9001\/embed\/exploration\/idToBeReplaced',
+    'http:\/\/localhost:9001\/assets\/scripts\/' +
+    'embedding_tests_dev_i18n_0.0.1.min.html - Refused to display ' +
+    '\'http:\/\/localhost:9001\/embed\/exploration\/idToBeReplaced\\?' +
+    'iframed=true&locale=en#version=0.0.1&secret=',
+    'http:\/\/localhost:9001\/assets\/scripts\/' +
+    'embedding_tests_dev_i18n_0.0.1.min.html - Refused to display ' +
+    '\'http:\/\/localhost:9001\/embed\/exploration\/fake_id//?' +
+    'iframed=true&locale=en#version=0.0.1&secret='
+  ];
```

Comment:
I still can't figure out why these errors are not ignored even after ignoring them. @U8NWXD @krishnarao22 Do you have any thoughts regarding this?